Sources/StreamChatUI/Components.swift

```diff
@@ -43,6 +43,9 @@ public struct Components {
     /// A button used for sending a message, or any type of content.
     public var sendButton: UIButton.Type = SendButton.self
 
+    /// A button used for recording a voice message,
+    public var recordButton: RecordButton.Type = RecordButton.self
```

Comment:
Where are the other common views? I guess they are in another PR? Would be nice to see the structure here, I'm especially interested in the MediaButton subclasses 👍

Reply:
The rest of the commonView are being introduced as part of this [PR](https://github.com/GetStream/stream-chat-swift/pull/2619). You can either look there for more context or in the `complete` PR [here](https://github.com/GetStream/stream-chat-swift/pull/2602).